static/app/views/monitors/monitors.tsx

```diff
@@ -195,7 +208,7 @@ const Filters = styled('div')`
 `;
 
 const StyledPanelTable = styled(PanelTable)`
-  grid-template-columns: 1fr max-content max-content max-content max-content max-content;
+  grid-template-columns: 1fr max-content 1fr max-content max-content max-content max-content;
```

Comment:
intentionally changed schedule column to 1fr to shorten width along with monitor name